README.md

```diff
@@ -2062,7 +2062,41 @@ describe('Admin View', function() {
   assert.equal(json.name, 'Daniel-san');
 ```
 
-### ChangeLog
-  - [Release Notes](/releases)
+### Upgrading
 
+#### v4 -> v5
```

Comment:
I'm being overly verbose here to make sure no-one is confused or left behind.

Realistically, the only changes needed are to add some deps, and import the factories.